packages/fds-components/src/style/interactive/buttonGroup.css

```diff
@@ -43,11 +43,6 @@
     border var(--button-group-timeout) var(--motion-easing-default);
 }
 
-.groupbtn .fds-icon {
```

Comment:
so unfortunately we can't remove this style until `fds-components` is updated to the `2.33` of `fds-icons` in a separate PR.

Reply:
got it. I know you explained this to me... I feel like I understood this last week, but need to wrap my head around it again. Reverting!